### PR TITLE
[DEV-10917] Remove the Covid-19 FABA index from Elasticsearch

### PR DESCRIPTION
**Pull request by aguest-kc**
**Description:**
Remove the Covid-19 FABA index from Elasticsearch.

**Technical details:**

**Requirements for PR merge:**

1. [x] Unit & integration tests updated
3. [x] Necessary PR reviewers:
    - [x] Backend
6. [x] Data validation completed
8. [x] Jira Ticket [DEV-10917](https://federal-spending-transparency.atlassian.net/browse/DEV-10917):
    - [x] Link to this Pull-Request

**Area for explaining above N/A when needed:**
```
2. API documentation updated
No API contracts are affected by this change.

4. Matview impact assessment completed
No matviews are affected by this change.

5. Frontend impact assessment completed
The frontend is not affected by this change.

7. Appropriate Operations ticket(s) created
No operations tickets are required for this change.
```

### CODE REVIEW

File: usaspending_api/search/v2/elasticsearch_helper.py [hunk 1]
```diff
@@ -3,14 +3,15 @@
 from typing import Dict, Optional
 
 from django.conf import settings
-from elasticsearch_dsl import A, Q as ES_Q
+from elasticsearch_dsl import A
+from elasticsearch_dsl import Q as ES_Q
```
Comment (dpb-bah) [inline usaspending_api/search/v2/elasticsearch_helper.py:7]: Any reason why these are split up?

**Review by dpb-bah — COMMENTED** [review]
Lots of deletions which is what the ticket says. Confirmed each of the functions/classes deleted are not used anywhere else in the branch.

I see the one spot where it updates to the DisasterBase. Does this cover all the bases?

File: usaspending_api/search/v2/elasticsearch_helper.py (hunk 1 above)
Comment (aguest-kc) [reply usaspending_api/search/v2/elasticsearch_helper.py:7]: Just a style change. It's easier to read when the `import as` is on a separate line.

**Comment by aguest-kc** [comment]
> I see the one spot where it updates to the DisasterBase. Does this cover all the bases?

Yeah, this is just one place that I missed in my previous branch where I updated the endpoints to use the new DB table instead of the ES index. The class needs to use `DisasterBase` so the `.as_view()` method works in the urls.py file.

**Review by dpb-bah — APPROVED** [review]

**Pull request merged**